src/neuroglancer/segmentation_display_state/frontend.ts

```diff
@@ -414,6 +420,20 @@ function makeRegisterSegmentWidgetEventHandlers(displayState: SegmentationDispla
     stickyChildren[template.visibleIndex].addEventListener('click', visibleCheckboxHandler);
     children[template.filterIndex].addEventListener('click', filterHandler);
     element.addEventListener('action:select-position', selectHandler);
+
+    const trackableRGB = new TrackableRGB(vec3.fromValues(0, 0, 0));
+    trackableRGB.changed.add(() => {
+      const testU = new Uint64(packColor(trackableRGB.value));
+
+      const idString = element.dataset.id!;
+      const id = tempStatedColor;
+      id.tryParseString(idString);
+      displayState.segmentStatedColors.value.delete(id);
```

Comment:
I'm curious why uint64_map doesn't trigger the changed signal if the value changes for an existing key. HashMapUint64 does not update values for existing keys. Not sure if those are both intentional.

Reply:
Yeah, HashMapUint64 should be fixed to update existing keys.  That is a bug that was not previously triggered because the hash map was only used for segment colors, which never changed after initial construction.